### ISSUE TITLE
[VoiceOver] Reader Post Detail: Links are not accessible

### ISSUE THREAD

**Issue by shiki** [wordpress-mobile/WordPress-iOS#12868]
<img src="https://user-images.githubusercontent.com/198826/68243306-76ebf880-ffcf-11e9-9b8a-3ea56b6178fe.png" width="320">

I cannot seem to navigate to links in the Post. Swiping right or left would just lead me to the next/previous paragraphs. There are also no custom actions (swipe up or down).

**Comment by leandroalonso** [comment]
Reader Detail now uses a webview, the issue doesn't happen anymore.
